src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
+        createFileAndWriteToFile(dataResult(readFromFile(fromFileName)), toFileName);
+    }
+
+    public String readFromFile(String fromFileName) {
+        StringBuilder inputString = new StringBuilder();
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            int character;
+            while ((character = bufferedReader.read()) != -1) {
+                inputString.append((char) character);
+            }
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException(e);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+        return inputString.toString();
+    }
+
+    public String dataResult(String data) {
+        int buy = 0;
+        int supply = 0;
+        String[] stringsFirst = data.split("\n");
+        for (String s : stringsFirst) {
+            String[] stringsSecond = s.split(",");
+            if (stringsSecond[0].equals("buy")) {
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/200301457-7ed8a07c-da02-445d-a51a-fa8d38a34d28.png)
Make 0, "buy" and 1 [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)